express/src/services/lambda-facade/StubLambdaFacade.ts

```diff
@@ -140,6 +153,10 @@ class StubLambdaFacade implements LambdaFacadeInterface {
     ): Promise<AxiosResponse> {
         return Promise.resolve({} as AxiosResponse);
     }
+
+    marshallArray(uris: string[]): Record<string, unknown> {
+        return {L: marshall(uris)};
+    }
```

Comment:
Import and use this method instead of the custom function - the SDK method automatically converts JavaScript objects to the list type `AttributeValue`
```
import {convertToAttr} from "@aws-sdk/util-dynamodb";
```